backend/business-partner-agent/src/main/java/org/hyperledger/bpa/impl/aries/AriesProofTemplateRequestBuilder.java

```diff
@@ -0,0 +1,64 @@
+package org.hyperledger.bpa.impl.aries;
```

Comment:
Move to prooftemplates to keep everything together.

Reply:
I moved it to  package org.hyperledger.bpa.impl.prooftemplates.aries;